### DISCUSSION TITLE
Include a couple of "standard" function

### DISCUSSION THREAD

**Discussion by olup** [wailsapp/wails#1914] [Ideas]
I might have missed it, and it already exists - but taking inspiration from Tauri and Electron, they provide a set of "out-of-the-box" functions so that js dev can use most native use-case without having to know much on the go language. Mostly : basic files manipulations (open, save, move, etc...), maybe some crypto - the scope would have to be defined. And this could be a separate module to include if needed.

And again, thanks for building this !

**Comment by leaanthony** [comment]
Please open an enhancement request for this if it doesn't already exist 👍
